applications/ConvectionDiffusionApplication/python_scripts/coupled_structural_thermal_solver.py

```diff
@@ -149,11 +150,23 @@ def Predict(self):
         self.thermal_solver.Predict()
 
     def SolveSolutionStep(self):
-        fluid_is_converged = self.structural_solver.SolveSolutionStep()
+        KratosMultiphysics.Logger.PrintInfo("\n" + "\t" +"Solving Structural part..." + "\n")
+        solid_is_converged = self.structural_solver.SolveSolutionStep()
+
+        self.RemoveConvectiveVelocity()
+
+        KratosMultiphysics.Logger.PrintInfo("\n" + "\t" + "Solving Thermal part..." + "\n")
         thermal_is_converged = self.thermal_solver.SolveSolutionStep()
 
-        return (fluid_is_converged and thermal_is_converged)
+        print("+--------------------------------------------------------------+")
+
+        return (solid_is_converged and thermal_is_converged)
 
     def FinalizeSolutionStep(self):
         self.structural_solver.FinalizeSolutionStep()
         self.thermal_solver.FinalizeSolutionStep()
+
+    def RemoveConvectiveVelocity(self):
+        if self.is_dynamic:
+            for node in self.thermal_solver.GetComputingModelPart().Nodes:
```

Comment:
USe VariableUtils, is faster

Reply:
Yes!! this was provisional. Now that you've mentioned, the variable utils can assign to the mesh velocity the velocity???? I was thinking in implementing the utility from scratch

Reply:
There are methods, let me check the name in python

Reply:
CopyModelPartNodalVar

Reply:
Checking... THX

Reply:
```suggestion
KratosMultiphysics.VariableUtils().CopyModelPartNodalVar(KratosMultiphysics.VELOCITY, KratosMultiphysics.MESH_VELOCITY, self.thermal_solver.GetComputingModelPart(), self.thermal_solver.GetComputingModelPart(), 0)
            
```